uk.co.compucorp.civicrm.tasksassignments/views/modal/assignment.html

```diff
@@ -128,18 +129,20 @@ <h4 class="modal-title">New Assignment</h4>
                       <input type="text" id="{{prefix}}assignment-task-due-{{$index}}" name="assignmentDue-{{$index}}" class="form-control" placeholder="{{ format }}" ng-model="activity.activity_date_time" uib-datepicker-popup is-open="dpOpened['task' + $index]" ng-required="!isDisabled && activity.create"
                           ng-click="!isDisabled && activity.create ? dpOpen($event, 'task'+$index) : return;" ng-disabled="isDisabled || !activity.create">
                         <span class="input-group-addon fa fa-calendar"></span>
-                        <span class="input-group-btn" ng-if="activity.isAdded">
-                          <a class="btn btn-default btn-danger" ng-click="removeActivity(taskList, $index)">
-                            <span class="fa fa-remove" aria-hidden="true"></span>
-                          </a>
-                        </span>
                     </div>
                   </td>
+                  <td align="center">
```

Comment:
Dont use non-css properties to style. Instead you can use bootstrap class to do the same.